### PR TITLE
redesigned like button to a sleeker heart. Added green/red to help in…

### PR DESCRIPTION
**Pull request by Slunk32**
…tuitive design for adding and removing to the playlist

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "camelcase": [0],
         "indent": [
             2,
-            "tab"
+            tab
```
Comment (amarcher) [inline .eslintrc]: Is tab defined?

File: public/js/components/Input.js [hunk 2]
```diff
@@ -50,18 +50,18 @@ const Input = React.createClass({
...
 			return (
-	      		<input
-	      			type="text"
-					ref={(element) => this.input = element}
-	      			className={this.props.className}
-	      			defaultValue={this.state.value}
-	      			onBlur={this.handleBlur}
-	      			onKeyDown={this.handleKeyDown} />
+				<input
+				type="text"
```
Comment (amarcher) [inline public/js/components/Input.js]: These should all be indented.

File: public/stylesheets/results.css [hunk 3]
```diff
@@ -52,21 +52,37 @@
 	overflow: hidden;
 }
 
-.like {
-	visibility: hidden;
-	background-color: #b7b7b7;
+
+.like-toggle.heart {
+	margin-right: 8px;
```
Comment (amarcher) [inline public/stylesheets/results.css:57]: common styles like `margin-right` can be in one `.heart {` syling clause.

File: src/index.js [hunk 4]
```diff
@@ -68,90 +68,90 @@ app.get('/recommended/:artist', function(req, res) {
...
-    spotifyApi.setAccessToken(authToken);
+	spotifyApi.setAccessToken(authToken);
 
-    spotifyApi.createPlaylist(userId, playlistName)
-        .then(resp => resp.body)
-        .then(playlist => {
-            res.status(200).send(playlist);
-        });
+	spotifyApi.createPlaylist(userId, playlistName)
+	.then(resp => resp.body)
```
Comment (amarcher) [inline src/index.js]: These should be indented

File: src/index.js [hunk 5]
```diff
@@ -68,90 +68,90 @@ app.get('/recommended/:artist', function(req, res) {
...
+	const userId = req.body.userId;
+	const playlistId = req.body.playlistId;
+	const playlistName = req.body.playlistName;
+
+	spotifyApi.setAccessToken(authToken);
+
+	spotifyApi.changePlaylistDetails(userId, playlistId, {
+		name: playlistName
+	})
+	.then(() => spotifyApi.getPlaylist(userId, playlistId))
```
Comment (amarcher) [inline src/index.js]: As should these

File: src/index.js [hunk 6]
```diff
@@ -68,90 +68,90 @@ app.get('/recommended/:artist', function(req, res) {
...
-        .catch(console.log);
+	const authToken = req.body.authToken;
+	const userId = req.body.userId;
+	const playlistId = req.body.playlistId;
+	const trackIds = req.body.trackIds;
+
+	spotifyApi.setAccessToken(authToken);
+
+	spotifyApi.addTracksToPlaylist(userId, playlistId, trackIds)
+	.then(() => spotifyApi.getPlaylist(userId, playlistId))
```
Comment (amarcher) [inline src/index.js]: And these

File: src/index.js [hunk 7]
```diff
@@ -68,90 +68,90 @@ app.get('/recommended/:artist', function(req, res) {
...
+	const userId = req.body.userId;
+	const playlistId = req.body.playlistId;
+	const trackIds = req.body.trackIds.split(',').map(trackId => {
+		return { uri: trackId };
+	});
+
+	spotifyApi.setAccessToken(authToken);
+
+	spotifyApi.removeTracksFromPlaylist(userId, playlistId, trackIds)
+	.then(() => spotifyApi.getPlaylist(userId, playlistId))
```
Comment (amarcher) [inline src/index.js]: And these

File: src/index.js [hunk 8]
```diff
@@ -68,90 +68,90 @@ app.get('/recommended/:artist', function(req, res) {
...
+	.catch(console.log);
 });
 
 app.get('/login', function(req, res) {
 	res.redirect('https://accounts.spotify.com/authorize' +
-	  '?response_type=token' +
-	  '&client_id=' + SPOTIFY_ID +
-	  '&scope=' + encodeURIComponent(SCOPES.join(' ')) +
-	  '&redirect_uri=' + encodeURIComponent(req.protocol + '://' + req.headers.host + REDIRECT_URI));
+	'?response_type=token' +
```
Comment (amarcher) [inline src/index.js]: And this

File: src/index.js [hunk 9]
```diff
@@ -161,17 +161,17 @@ app.get('/authenticate', function(req, res) {
...
 
-    spotifyApi.getMe()
-        .then(user => {
-            res.status(200).send(user.body);
-        })
-        .catch(error => {
-            console.log(error);
-        });
+	spotifyApi.getMe()
+	.then(user => {
```
Comment (amarcher) [inline src/index.js]: And these

**Comment by Slunk32** [comment]
I fixed some of the spacing stuff, let me know if we should merge this

**Pull request merged**